extensions/wikia/JWPlayerTag/JWPlayerTagController.class.php

```diff
@@ -38,6 +38,7 @@ public function renderTag( $input, array $args, Parser $parser, PPFrame $frame )
 			RequestContext::getMain()->getTitle()->getPrefixedDBkey()
 		) ) {
 			$script = JSSnippets::addToStack( [
+				'jwplayer_tag_ads_js',
```

Comment:
I guess it will cause loading some javascript twice:
`'//extensions/wikia/AdEngine/js/video/player/jwplayer/jwplayerAdsTracking.js',
'//extensions/wikia/AdEngine/js/video/player/jwplayer/jwplayerTracker.js',`

Reply:
[The bug](https://wikia-inc.atlassian.net/browse/ADEN-6383) was exactly opposite: JW Player instances generated by `<jwplayer>` tag don't work on featured video pages because there are no ads assets loaded (featured video JW Player instance works). By adding this group here we fix it. If it's not a featured video page it goes to the `else` block and loads two packages. The JS package includes `jwplayer_tag_ads_js` in that case. We still can think of a cleaner way for loading those assets, though.